pytket/extensions/quantinuum/backends/federated_login.py

```diff
@@ -34,19 +34,19 @@ def microsoft_login() -> Tuple[str, str]:
     # check if the device code is available in the flow
     if "user_code" not in flow:
         raise ValueError(
-            "Fail to create device flow. Err: %s" % json.dumps(flow, indent=4)
+            f"Fail to create device flow. Err: {json.dumps(flow, indent=4)}"
         )
 
     # this prompts the user to visit https://microsoft.com/devicelogin and
     # enter the provided code on a separate browser/device
     code = flow["user_code"]
     link = flow["verification_uri"]
 
-    print("To sign in:")
-    print("1) Open a web browser (using any device)")
-    print("2) Visit " + link)
-    print("3) Enter code '" + code + "'")
-    print("4) Enter your Microsoft credentials")
+    print("To sign in:")  # noqa: T201
```

Comment:
I was not fully sure which prints to keep, and which to remove. I think some of them might be helpful, but I removed some of them. Happy to discuss this and add the removed prints back in.